xcm/xcm-simulator/fuzzer/README.md

```diff
@@ -0,0 +1,34 @@
+# XCM Simulator Fuzzer
+
+This project will fuzz-test the XCM simulator. It can catch reachable panics, timeouts as well as integer overflows and underflows.
+
+## Install dependencies
+
+```
+cargo install honggfuzz
+```
+
+## Run the fuzzer
+
+```
+cargo hfuzz run xcm-fuzzer
```

Comment:
nit: This should say that we have to be in this directory. It doesn't work from root.

Reply:
Fixed :heavy_check_mark: